content/millennium/r4/diagnostic/observation.md

```diff
@@ -0,0 +1,122 @@
+* [Component (eg: systolic and diastolic for blood pressure)](https://hl7.org/fhir/R4/observation-definitions.html#Observation.component){:target="_blank"}
+
+## Terminology Bindings
+<%= terminology_table(:observation, :r4) %>
+
+## Extensions
+
+* valueAttachment: URL for this extension is defined as: `https://hl7.org/fhir/5.0/StructureDefinition/extension-Observation.valueAttachment`
+  This extension is defined and referenced from the newer version of FHIR. See [Extensions for converting between versions] and [R5 Snapshot of Observation.value] for more information.
+
+## Search
+
+Search for labs, vitals, and alcohol/tobacco use Observations that meet supplied query parameters:
+
+    GET /Observation?:parameters
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+_Implementation Notes_
+
+* The `comments` field may have RTF or other formatted data rather than simple text.
+
+* When multiple pages of Observation results are returned for a single query:
+  * All Social history Observations (if any qualify for the query) will be returned on the first page of results. This means that the next bullet does not apply to Social history Observations.
+  * Results are sorted by effective date/time in descending order by page. That is, all Observations on any given page of results are newer than all Observations on the next page of results. Sort order within pages is not guaranteed.
+  * If the query uses the `_lastUpdated` query parameter, results are sorted by last updated date/time in descending order by page, not by effective date/time.
+
+### Parameters
+
+ Name             | Required?         | Type          | Description
+------------------|-------------------|---------------|-----------------------------------------------------------------------------------------------------------------------------------------
+ `patient`        | This or `subject` | [`reference`] | The subject that the observation is about (if patient). Example: `patient=1316024`
+ `subject`        | This or `patient` | [`reference`] | The subject (Patient) that the observation is about. Example: `subject=Patient/1316024` or `subject:Patient=1316024`
+ `code`           | N                 | [`token`]     | The code or component-code of the observation type. Example: `code=http://loinc.org|3094-0,http://loinc.org|3139-3`
+ `date`           | N                 | [`date`]      | Date range into which the observation falls. Example: `date=gt2014-09-24` or `date=lt2015-09-24T12:00:00.000Z`
+ `_lastUpdated`   | N                 | [`date`]      | Date range in which the observation was last updated. Example: `_lastUpdated=gt2014-09-24` or `_lastUpdated=lt2015-09-24T12:00:00.000Z`
+ `category`       | N                 | [`token`]     | The category of observations. Example: `category=laboratory`
+ [`_count`]       | N                 | [`number`]    | The maximum number of results to return per page. Defaults to `50`.
```

Comment:
Remove the default value. We don't need to advertise that.

Reply:
https://github.com/cerner/fhir.cerner.com/pull/345/commits/3756e931418b52a36c0e6998015daa0d7adbb1c5#diff-d42a2575832583fdd3d20dff4a54f294L75